specs/latest/2.0/index.html

```diff
@@ -3117,6 +3117,18 @@ <h2 class="no-toc">Table of contents</h2>
         implementations.
     </div>
 
+    <h3>Framebuffer contents after invalidation</h3>
+    <p>
+        In OpenGL ES 3.0, after calling <code>invalidateFramebuffer</code> or <code>invalidateSubFramebuffer</code>,
+        the affected region's contents are undefined. In WebGL 2.0, it is required for the contents to either stay
+        unchanged or become all 0.
```

Comment:
I understand where you come from, but "reflect the results of subsequent rendering operations" isn't really a state right after the invalidation call. When the spec defines "become all 0", it is really not asking implementations to insert a clear() call, but rather, only to set up a bit indicating the content is undefined.  Only when this undefined content is about to be exposed to a user, then we need to do some clearing.

All other suggested changes are good and I'll get a pull request to incorporate them.